client/app/assets/styles/variables.js

```diff
@@ -7,12 +8,15 @@ const textColor = 'rgb(82, 89, 97)';
 const textColorFocus = 'rgb(0, 0, 0)';
 const backgroundLightColor = 'white';
 const backgroundLightColorHover = 'rgba(169, 172, 176, 0.07)';
-const paddingTopbarVertical = '0.92857em';
-const paddingTopbarHorizontal = '1.7142857em';
 
 const topbarBorderColor = 'rgba(169, 172, 176, 0.5)';
 const bodyPadding = '18px';
 
+const pxToEms = function pxToEms(px, againstFontSize) {
+  const emValue = px / againstFontSize;
+  return `${emValue}em)`;
+};
```

Comment:
This should go e.g. to `client/app/assets/styles/functions.js` and in use with `postcss-functions` in the future so it can be used in all CSS files, but fine here for now.